sr_multi_moveit/sr_box_ur10_moveit_config/config/robot_configs/right_ur10.yaml

```diff
@@ -2,7 +2,7 @@ robot:
   name: ursrbox
   manipulators:
     right_manipulator:
-      side: right
+      side: left
```

Comment:
left? this is right_ur10.yaml

Reply:
Fixed in https://github.com/shadow-robot/sr_interface/pull/419/commits/69bc13be8e64f769c2179bda3d2bfc388b4bc071